.github/workflows/ci.yml

```diff
@@ -58,7 +58,7 @@ jobs:
 
     services:
       postgres:
-        image: postgis/postgis:12-2.5
+        image: postgis/postgis:16-3.5
```

Comment:
In docker-compose we have postgres 17. Should we have the same version here and in the code-quality job?

Reply:
good point, I'll use 17 here as well